shared/fs/container.js

```diff
@@ -1,16 +1,57 @@
 // @flow
+import * as I from 'immutable'
 import {connect, type TypedState, type Dispatch} from '../util/container'
 import * as FSGen from '../actions/fs-gen'
-import Fs from '.'
+import Files from '.'
+import {navigateAppend, navigateUp} from '../actions/route-tree'
+import * as Types from '../constants/types/fs'
 
-const mapStateToProps = (state: TypedState) => ({
-  counter: state.fs.counter,
-  you: state.config.username,
+type StateProps = {
```

Comment:
i don't think we actually need these types to be explicit in the return values. flowtyping of connect has gotten a lot better and it should be inferable